examples/dt/osasinfra/deployment/kustomization.yaml

```diff
@@ -0,0 +1,13 @@
+# This is the kustomization for the FINAL step, edpm-post-ceph
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+components:
+  - ../../../../dt/osasinfra/edpm-post-ceph/deployment
+  # - https://github.com/openstack-k8s-operators/architecture/va/hci/edpm-post-ceph/?ref=main
+  ## It's possible to replace ../../../va/hci/edpm-post-ceph/ with a git checkout URL as per:
```

Comment:
This comment is referencing VA1 and should probably be updated for this DT

Reply:
Indeed, I just pushed an update where I fixed all occurrences.